.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -303,3 +303,24 @@ swap:
   wish: want
   wish|would like: want
   zero out: zero
+  byte code: bytecode
+  client: customer
+  computer farm|computer ranch: server cluster
+  details pane: details view
+  domain mode: managed domain
+  enum: data enumeration
+  live broker: master broker
+  live only: live-only
+  load-balance|load-balancing: load balance
+  multi site|multi-site: multisite
+  passive broker: slave broker
+  pojo: data model
+  regex: regular expression
+  snapset|snapsets: snapshot set
+  standalone mode: standalone server
+  sub version|subversion: sub-version
```

Comment:
Not sure if it matters in our docs, but this rule might flag instances of Apache Subversion.

Reply:
updated